src/pybind/mgr/orchestrator/module.py

```diff
@@ -38,6 +39,20 @@ class OrchestratorCli(OrchestratorClientMixin, MgrModule):
                              'test_orchestrator'],
             'runtime': True,
         },
+        {
+            'name': 'grafana-tls-cert',
+            'type': 'str',
+            'default': None,
+            'desc': 'Grafana SSL/TLS certificate',
+            'runtime': True,
+        },
+        {
+            'name': 'grafana-tls-key',
+            'type': 'str',
+            'default': None,
+            'desc': 'Grafana SSL/TLS private key',
+            'runtime': True,
+        },
```

Comment:
...and if we put it under mgr/cephadm, i think we should either (1) move these comands to 'ceph cephadm ...' and cephadm/module.py, or just document using the raw config-key commands, e.g.,
```
ceph config-key set mgr/cephadm/grafana_crt -i crt
ceph config-key set mgr/cephadm/grafana_key -i key
```
...which might be the easiest thing.  I'm not sure the friendly CLI commands are adding that much to the experience?

Reply:
I removed the CLI commands for now although I think they might be useful at some point in the future. But it definitely simplifies the code required to have this feature implemented and that suffices me.

But I also noticed something unexpected. I need to restart the mgr module for it to reload the configuration change. If I set the grafana cert and key, it won't be able to read it unless restarted. Is it simply how `self.set_store()` works?

If you don't mind, I'd wait with documenting the configuration option until https://github.com/ceph/ceph/pull/33700 is merged, as your PR adds a monitoring section to the cephadm documentation I'd simply extend.